### ISSUE TITLE
Error when installing ALB ingress for datahub frontend

### ISSUE THREAD

**Issue by hugh-nguyen** [acryldata/datahub-helm#101]
**Describe the bug**
I'm following the guide to setting up DataHub on AWS
https://datahubproject.io/docs/deploy/aws

I've gotten to the stage where I need to update values.yaml to in order to attach the ALB ingress to the frontend.

It looks like this -

```
datahub-frontend:
  enabled: true
  image:
    repository: linkedin/datahub-frontend-react
    tag: "latest"
  ingress:
    enabled: true
    annotations:
      kubernetes.io/ingress.class: alb
      alb.ingress.kubernetes.io/scheme: internet-facing
      alb.ingress.kubernetes.io/target-type: instance
      alb.ingress.kubernetes.io/certificate-arn: arn:aws:acm:ap-southeast-2:601628467906:certificate/xxxxxx
      alb.ingress.kubernetes.io/inbound-cidrs: 0.0.0.0/0
      alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80}, {"HTTPS":443}]'
      alb.ingress.kubernetes.io/actions.ssl-redirect: '{"Type": "redirect", "RedirectConfig": { "Protocol": "HTTPS", "Port": "443", "StatusCode": "HTTP_301"}}'
    hosts:
      - host: xxxx
        redirectPaths:
          - path: /*
            name: ssl-redirect
            port: use-annotation
        paths:
          - /*
```

When I run the upgrade however I am getting the following error.
```
Error: UPGRADE FAILED: error validating "": error validating data: ValidationError(Ingress.spec.rules[0].http.paths[0].backend.service.port.number): invalid type for io.k8s.api.networking.v1.ServiceBackendPort.number: got "string", expected "integer"
```

**Expected behavior**
Successful Upgrade

**Additional context**
ALB seems to there, and site is reachable when using port forwarding for local access

**Comment by krishnakumartm** [comment]
Did you tried changing "Port": "443" to "Port": 443 in alb.ingress.kubernetes.io/actions.ssl-redirect

**Comment by hugh-nguyen** [comment]
Hi,

There's an issue with version 0.2.63 of the datahub helm chart.

I installed 0.2.62 and it worked.

helm install datahub datahub/datahub --values values.yaml --version 0.2.62

OR 

helm upgrade --install datahub datahub/datahub --values values.yaml --version 0.2.62

Hope this helps

**Comment by felixb** [comment]
there is a proposed fix in #100.
